README.md

```diff
@@ -11,9 +11,36 @@
 
 🐺 φεγγάρι - The Lua VM written in JS ES6 for Node and the browser
 
+## Getting started in the browser
+
+Fengari is not released yet but you can experiment with it in the browser by following the instructions of the [fengari-web](https://github.com/fengari-lua/fengari-web) repository.
+
 ## Semantics
 
-Fengari implements Lua 5.3 semantics and will hopefully follow future Lua releases. If you find any noticeable difference between Fengari and Lua's behaviours, please [report it](https://github.com/fengari-lua/fengari/issues).
+This repository contains the core fengari code (which is a port of the Lua C library) which includes parser, virtual machine and base libraries.
```

Comment:
This doesn't belong under `Semantics` (should be higher)

Reply:
oh this is a rebasing error